synapse/models/person.py

```diff
@@ -1,41 +1,238 @@
+                        'doc': 'The English version of the given name of the person'
+                    }),
+                )),
+                ('ps:persona', {}, (
+                    # FIXME aliases?
+                    # ('guidname', ('str', {'lower': True}), {
+                    #     'doc': 'The GUID resolver alias for the suspected person.',
+                    # }),
+                    ('dob', ('time', {}), {
+                        'doc': 'The Date of Birth (DOB) if known.',
+                    }),
+                    # FIXME need file:bytes
+                    # ('img', ('file:bytes', {}), {
+                    #     'doc': 'The primary image of a suspected person.'
+                    # }),
+                    ('nick', ('inet:user', {}), {
+                        'doc': 'A username commonly used by the suspected person',
+                    }),
+                    ('name', ('ps:name', {}), {
+                        'doc': 'The localized name for the suspected person.',
+                    }),
+                    ('name:sur', ('ps:tokn', {}), {
+                        'doc': 'The surname of the suspected person.'
+                    }),
+                    ('name:middle', ('ps:tokn', {}), {
+                        'doc': 'The middle name of the suspected person.'
+                    }),
+                    ('name:given', ('ps:tokn', {}), {
+                        'doc': 'The given name of the suspected person.'
+                    }),
+                    ('name:en', ('ps:name', {}), {
+                        'doc': 'The English version of the name for the person.',
+                    }),
+                    ('name:en:sur', ('ps:tokn', {}), {
+                        'doc': 'The English version of the surname of the suspected person.'
+                    }),
+                    ('name:en:middle', ('ps:tokn', {}), {
+                        'doc': 'The English version of the middle name of the suspected person.'
+                    }),
+                    ('name:en:given', ('ps:tokn', {}), {
+                        'doc': 'The English version of the given name of the suspected person.'
+                    }),
+                )),
+                ('ps:person:has', {}, (
+                    ('person', ('ps:person', {}), {
+                        'ro': True,
+                        'doc': 'The person who owns or controls the object or resource.',
+                    }),
+                    ('node', ('ndef', {}), {
+                        'ro': True,
+                        'doc': 'The object or resource that is owned or controlled by the person.',
+                    }),
+                    ('node:form', ('str', {}), {
+                        'ro': True,
+                        'doc': 'The form of the object or resource that is owned or controlled by the person.',
+                    }),
+                    # FIXME Add seen:min and seen:max
```

Comment:
Are we going to keep seen:min/max? I thought these were going to become universal props

Reply:
i think we can drop all these comments when that lands.